go.mod

```diff
@@ -56,6 +56,8 @@ require (
 	golang.org/x/term v0.3.0
 )
 
+require github.com/docker/go-units v0.5.0 // indirect
```

Comment:
You don't need to use newer libraries here @shtripat you can use `humanize.IBytes()`

Reply:
Oh ok. Let me do that. Thanks for suggestion!

Reply:
@harshavardhana only thing is `humanize.IBytes` or `humanize.BigBytes` return a value rounded to int e.g. 41.49 MiB rounded as 42 MiB. Dont feel that would look very optional. Suggestions?

Reply:
We use `humanize.IBytes` everywhere @shtripat so that is the correct approach.